lib/ut/fold.c

```diff
@@ -43,7 +43,7 @@ void test_fold(void)
 	/* Test summation. */
 	M0_UT_ASSERT(m0_reduce(i, 4, 0, + a[i]) == a[0] + a[1] + a[2] + a[3]);
 	/* Test empty range. */
-	M0_UT_ASSERT(m0_reduce(i, 0, 8, + 1/x) == 8);
+	M0_UT_ASSERT(m0_reduce(i, 0, 8, + 1) == 8);
```

Comment:
This will fix the Codacy warning, but this will miss the purpose of this test.

This is to test "empty range". the "+1/x" will never be called.

For example, we can use:
`+ 1 /(a[1] -1)` to replace this. I hope the Codacy static checking will not produce the warning again.

Reply:
Okay. I will use `a[2]-1` as denominator to have non-zero and different value from numerator. Otherwise codacy reports the info/warning if both are same.

Reply:
Done.

Reply:
Looks good.

Reply:
We can add a comment to say that:
We are purposely to make the denominator to be zero.

Reply:
If we want to make sure that the expression is not executed then as you say above we can change to '+ 1' (or even '+ 1/x') so that if the expression is evaluated and used in the equation then the check '== 8' will fail and M0_UT_ASSERT will trigger a crash.